packages/core-flows/src/definition/cart/utils/prepare-confirm-inventory-input.ts

```diff
@@ -0,0 +1,61 @@
+import { ProductVariantDTO } from "@medusajs/types"
+import { MedusaError } from "medusa-core-utils"
+
+interface ConfirmInventoryPreparationInput {
+  productVariantInventoryItems: {
+    variant_id: string
+    inventory_item_id: string
+    required_quantity: number
+  }[]
+  items: { variant_id?: string; quantity: number }[]
+  variants: ProductVariantDTO[]
```

Comment:
todo: it seems that we dont need the DTO but only the id and manage inventory, if thats correct it would be better to only requires those instead of requiring the entire dto. This should always be the approach to take IMO